### PR TITLE
Graderutils shell script fix and readme

### PR DESCRIPTION
**Pull request by ihalaij1**
**What?**

- Arguments ordering no longer matters when using graderutils command
- Readme updated with graderutils command arguments

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,10 +29,25 @@ Utility commands
...
+  * `--novalidate`
+    * Skip validation of test config
+  * `--container`
+    * This flag should be used when running graderutils inside docker container based on apluslms/grading-base
+  * `--show-config`
+    * Print test configuration into warnings.
+  * `--develop-mode`
+    * Display all unhandled exceptions unformatted.
+    Also implies `--show-config`.
+    By default, exceptions related to improperly configured tests are catched and hidden behind a generic error message.
```
Comment (Mankro) [inline README.md:46]: English: catched -> caught

**Review by Mankro — APPROVED** [review]
I will merge this. I was wondering about the `--container` flag. Is it necessary to always define it in the exercise config.yaml since grading is normally always run in a container?

**Comment by ihalaij1** [comment]
It seems that the `--container` flag seems to only matter in cases where Graderutils works incorrectly and raises an exception.
I will test this and add the flag by default if necessary.

**Comment by ihalaij1** [comment]
The `--container` flag may be remnant of some old code, because if it is set to `True`, grader-utils actually works incorrectly (as in a manner that is not desired) in A+ and doesn't provide any visible feedback. Instead the exception traceback is printed to stderr:
https://github.com/apluslms/python-grader-utils/blob/4845b37b3bae08ee7b473ce9fea6942981bc3e02/graderutils/main.py#L151-L153

What do you think we should do to it? I don't think the flag should ever be set to `True` in normal circumstances.

**Comment by Mankro** [comment]
Right, well, let's leave it as is for the time being. But the README should then say clearly that the flag is not normally used.

**Comment by ihalaij1** [comment]
Yeah, I can change the README (and fix that one English spelling mistake).
The current description was taken from grader-utils. It should be changed there too.

**Pull request merged**